decidim-core/app/presenters/decidim/resource_locator_presenter.rb

```diff
@@ -70,6 +70,16 @@ def edit(options = {})
       admin_route_proxy.send("edit_#{member_route_name}_path", target, options)
     end
 
+    # Generates and admin url only if the manifest has the property :admin_route_name defined
+    # this allows to distinct from resources that can be administrated from those that are not
+    #
+    # options - An optional hash of options to pass to the Rails router
+    #
+    # Returns a String.
+    def admin_url(options = {})
+      admin_member_route("url", options.merge(host: root_resource.organization.host))
+    end
```

Comment:
@andreslucena could you give me feedback regarding this? I've seen that in other places we create admin urls using a different approach: https://github.com/decidim/decidim/blob/develop/decidim-proposals/app/events/decidim/proposals/admin/proposal_note_created_event.rb#L15

I'm guessing that the approach in this PR could be useful for the future. If you think this is ok I'll bring in the specs for this.